src/Symfony/Component/HttpClient/Response/CurlResponse.php

```diff
@@ -345,6 +345,10 @@ private static function parseHeaderLine($ch, string $data, array &$info, array &
                 return 0;
             }
 
+            if (isset($headers['content-type'], $options['buffer']) && \is_string($options['buffer'])) {
```

Comment:
`if (!$content && [...]`

Reply:
Changed the check. Should I change the order of the conditions too?